src/miner.cpp

```diff
@@ -123,7 +123,7 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
     CAmount nFees = 0;
 
     {
-        LOCK2(cs_main, mempool.cs);
+        LOCK(cs_main);
```

Comment:
I don't see how any of changes here (miner.cpp) are related to this PR. Should be reverted imo.